app/lib/tax_return_status.rb

```diff
@@ -43,18 +43,18 @@ def message_templates
 
       review_ready_for_qr: 301, review_reviewing: 302, review_ready_for_call: 303, review_signature_requested: 304, review_info_requested: 305,
 
-      file_ready_to_file: 401, file_efiled: 402, file_mailed: 403, file_rejected: 404, file_accepted: 405, file_not_filing: 406
+      file_ready_to_file: 401, file_efiled: 402, file_mailed: 403, file_rejected: 404, file_accepted: 405, file_not_filing: 406, file_hold: 407
```

Comment:
Not a blocker, but when we are adding statuses we could choose to allow for more values in between. For example, this could be `470` instead of `407`. That way we could add more statuses before or after the new status easily in the future.